src/constants/gameList.ts

```diff
@@ -0,0 +1 @@
+export const GAME_LIST = ['LEAGUE OF LEGENDS', 'BATTLEGROUNDS', 'OVERWATCH 2', 'MINCRAFT'];
```

Comment:
```suggestion
export const GAME_LIST = ['LEAGUE OF LEGENDS', 'BATTLEGROUNDS', 'OVERWATCH 2', 'MINECRAFT'];
```
게임 이름에 오타가 있는 것 같습니다.

Reply:
수정했습니다!